Makefile

```diff
@@ -0,0 +1,75 @@
+# Copyright 2016 The Kubernetes Authors.
```

Comment:
In both service-catalog and OpenShift we do a dumb regen on every changed go file under apis and it seems to work fine; I doubt all the makefile complexity is worth it at all.